main/cmd/initializer.go

```diff
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"container/list"
+
+	"ses.genesis.com/exchange-web-service/main/service"
+
+	"ses.genesis.com/exchange-web-service/main/application/exchange"
+	"ses.genesis.com/exchange-web-service/main/application/exchange/provider"
+	"ses.genesis.com/exchange-web-service/main/application/notification"
+	persistent2 "ses.genesis.com/exchange-web-service/main/persistent"
```

Comment:
```suggestion
	"ses.genesis.com/exchange-web-service/main/persistent"
```
And then use `persistent`

Reply:
Done